Makefile

```diff
@@ -19,3 +19,51 @@ test:
 format:
 	dart format --output=write .
 
+ifeq ($(OS),Windows_NT)
+	OS_NAME := Windows
+else
+	OS_NAME := $(shell uname -s)
+endif
+
+GOOGLE_IOS_FILE := GoogleService-Info.plist
+GOOGLE_ANDROID_FILE := google-services.json
+ifeq ($(OS_NAME),Windows)
+	IOS_DEST_PATH := ios\Runner
+	IOS_SRC_PATH := \config\$(GOOGLE_IOS_FILE)
+	ANDROID_DEST_PATH := android\app
+	ANDROID_SRC_PATH := \config\$(GOOGLE_ANDROID_FILE)
+else
+	IOS_DEST_PATH := ios/Runner
+	IOS_SRC_PATH := config/$(GOOGLE_IOS_FILE)
+	ANDROID_DEST_PATH := android/app
+	ANDROID_SRC_PATH := config/$(GOOGLE_ANDROID_FILE)
+endif
+
+CLEAN_COMMAND := flutter clean
+BUILD_COMMAND := flutter build ios --simulator
```

Comment:
Isn't this hardcoded to IOS builds in this way?

Reply:
yeah, actually it isn't right, not just because I use it for the android goal as well :DD but because there is no need for an ios build goal for Windows since that is not possible on that platform, therefore I'll remove that condition and update the android command as well accordingly, but thank you for the notice!